requests-for-comments/2025-06-10-geo-and-llms-txt.md

```diff
@@ -0,0 +1,116 @@
+
+The name of the game is to influence what information LLMs retrieve, especially through their *web search* and *deep research* features, for the prompts we care about. But what sources do they trust and cite during web retrieval? And what kind of content?
+
+*Spoiler*: It's a lot of channels.
+- Our own web pages
+- Traditional SERP, top 20 results on Google
+- Press, PR articles, and journalistic outlets like techcrunch and forbes
+- Organic discussions on Reddit or other community boards 
+- Third-party blogs
+- Crappy syndicated posts like "Top 5 tools for X" on sites like dev.to
+- YouTube
+- Wikipedia
+- And the list goes on...
+
+AI web search pulls from a broad and fuzzy pool of sources. What gets prioritized depends on the prompt, the industry, and other external factors (e.g., [OpenAI partnering with Reddit](https://openai.com/index/openai-and-reddit-partnership/) or [Perplexity partnering with Bing](https://www.hyperlinker.ai/en/seo/perplexity-ai-bing)). It's basically a black box. 
+
+But wait! There are fancy new GEO tools that track branded mentions across LLMs, so we can measure our AI "rank" or "visibility" for the prompts we're targeting. We just started using one called [Gauge](https://withgauge.com/).
+
+## Okay, so what actually changes?
+
+In terms of creating content, not much. Content still has to be [great](https://newsletter.posthog.com/i/156101009/your-great-content-probably-sucks), and there's no easy hack for that.
+
+But where things *do* change is in distribution and coverage. 
+
+In GEO, our primary goal is to have our brand, content, or insights included in the AI's synthesized response. This means we need more content in more places for the LLMs to retrieve and reference.
+
+For example, in the context GEO, it's more valuable for PostHog to appear across multiple spots in the top 20 Google results than to only be mentioned just once in the #1 spot. 
+
+This also implies more third-party content. Think:
+- Traditional PR and press (TechCrunch, Forbes, etc.)
+- Partnerships and syndication (e.g. comparison articles on dev.to or newsletters)
+- Youtube content
+- Community discussion on forums like Reddit
+
+![PR is important again 😂](https://res.cloudinary.com/dmukukwp6/image/upload/SCR_20250611_lqlo_3ca5da33be.png)
+
+## A note on training vs. retrieval
+
+Our content can appear in AI responses through either pathway: by being part of the model's training data or by being dynamically retrieved during a web search operation or other RAG methods. 
+
+GEO focuses on the second path, by influencing the LLM's diet instead of its training. In web retrieval, LLMs act more like reasoning engines, pulling from the web, distilling a bunch of information, and providing a structured answer with citations.
+
+## Some good reads
+- https://a16z.com/geo-over-seo/
+- https://www.mbcrosier.com/posts/ai-search-optimization
+- https://www.ignorance.ai/p/seo-for-ai-a-look-at-generative-engine
+
+## Solutions
```

Comment:
Find a way to run Perplexity ads? I'm not sure if they have an official product or not: https://www.perplexity.ai/hub/blog/why-we-re-experimenting-with-advertising